RELEASING.md

```diff
@@ -23,6 +32,14 @@ to release in the input field that pops up, and then press "Run workflow".
 This triggers the release process, which builds the artifacts, publishes the artifacts, and creates
 and pushes a git tag with the version number.
 
+After making the release:
+
+* Update the version (bump and add `-SNAPSHOT`) in these files:
+  * build.gradle.kts
+  * gradle-plugins/build.gradle.kts
```

Comment:
Nit: WDYT about keeping the version in some `version.properties` file that could be reused by all projects involved in the composite build?